app/import_functions.go

```diff
@@ -1159,64 +1161,74 @@ func (a *App) getChannelsForPosts(teams map[string]*model.Team, data []*PostImpo
 	return channels, nil
 }
 
-func (a *App) importMultiplePosts(data []*PostImportData, dryRun bool) *model.AppError {
-	if len(data) == 0 {
-		return nil
+// getPostStrID returns a string ID composed of several post fields to
+// uniquely identify a post before it's imported, so it has no ID yet
+func getPostStrID(post *model.Post) string {
+	return fmt.Sprintf("%d%s%s", post.CreateAt, post.ChannelId, post.Message)
+}
+
+func (a *App) importMultiplePostLines(lines []LineImportWorkerData, dryRun bool) (*model.AppError, int) {
```

Comment:
Maybe we can add a comment and mention that this will return the error line in case of an error. Also, would `(int, *model.AppError)` be more idiomatic? Not so sure btw, this may contradict with general use case of `(int, err)`.

Reply:
My rationale not to invert the order is that the error line is information relative to the error, kind of "on top of" the error (you check the error first and if it's present, then you check the line) but I understand how it can go against a common pattern, so I'm happy making the change.

Reply:
Fixed in 90f033a